### ISSUE TITLE
Spike: Try out a different MarkDown provider for the HTML output format

### ISSUE THREAD

**Issue by dirkrombauts** [picklesdoc/pickles#269]
Try out [Strike.Jint.Merged](https://www.nuget.org/packages/Strike.Jint.Merged/) as the MarkDown provider, so that we can optionally enable GitHub Flavored Markdown later.
- [ ] Add it to the code.
- [ ] Make sure the unit tests are still passing
- [ ] Generate sample outputs for Html, and compare them to the sample output on the pickles website.

**Comment by dirkrombauts** [comment]
@dstj would you have time to take a stab at this?

**Comment by dstj** [comment]
Hi Dirk, it won't be possible in the next few days/weeks,but I'll try to see if I can make up some time to try it out in a consistent manner. I'll like to avoid having my own separate custom fork for that...

**Comment by dirkrombauts** [comment]
@dstj I'm playing around with this but I'm running into problems. Did you ever use Strike/markdownify in the static HTML version or only in the dynamic html version?

It works for me in the dynamic html version but in the static html version my own code is unhappy about `<img>` elements not being selfclosed.

**Comment by dstj** [comment]
@dirkrombauts, I am unsure I understand your question, I don't recall seeing a dynamic or static option. My usage was very simplistic. I used it to transform Markdown file to HTML and the only code I need was:

```
public class MarkdownifyProvider : IMarkdownProvider
	{
		private readonly Markdownify markdownify;

		public MarkdownifyProvider()
		{
			this.markdownify = new Markdownify();
		}
		
		public string Transform(string text)
		{
			var transform = markdownify.Transform(text);
			transform = transform.Replace("&nbsp;", string.Empty);
			return transform;
		}
	}
```

**Comment by dirkrombauts** [comment]
@dstj no problem, I found the solution for my problem. In fact there was no problem in Markdownify but rather in its documentation.

**Comment by dirkrombauts** [comment]
Released as an experimental feature in version [2.5.0](https://github.com/picklesdoc/pickles/releases/tag/v2.5.0).

**Comment by dirkrombauts** [comment]
It's been an amazing time for me to work on Pickles. Now it's finally time for me to lay down the mantle and move on. I am leaving Pickles completely.

I am closing this issue, so that the next maintainer of this repository can start from a clean slate.

Do you want to take over active development and maintenance at Pickles? Contact me directly at dirk dot rombauts at picklesdoc dot com. I will hand over everything Pickles-related to you. This email address will remain active until 11 December 2020.
